### PR TITLE
Multitoken tests

### PR DESCRIPTION
**Pull request by jakcinmarina**

### CODE REVIEW

**Review by vlnevyhosteny — CHANGES_REQUESTED** [review]

File: packages/tatum-bsc/src/transaction/bsc.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ import {
   UpdateCashbackErc721,
   validateBody,
   obtainCustodialAddressType,
+  erc1155TokenABI,
```
[the remark this answers is not in the document]
Comment (jakcinmarina) [reply packages/tatum-bsc/src/transaction/bsc.ts:62]: yeah it was the reason tests failed for most chains. 
erc1155TokenABI was often used in code but wasn't imported at all

File: packages/tatum-one/src/transaction/one.ts [hunk 2]
```diff
@@ -26,6 +26,7 @@ import {
   obtainCustodialAddressType,
   ChainOneMint721,
   OneMint721,
+  erc1155TokenABI,
```
[the remark this answers is not in the document]
Comment (jakcinmarina) [reply packages/tatum-one/src/transaction/one.ts:29]: wasn't initially imported but was used in code

File: packages/tatum/src/multiToken/tests/celo.spec.ts [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+  it('should deploy multitoken', async () => {
+    try {
+      const body = {
+        chain: Currency.CELO,
+        fromPrivateKey: '0d6c13fe5fed644dfa02512d4bffde9453dcb48873afb0b0a4c0cebce160c279',
+        uri: 'tatum-test',
+        feeCurrency: Currency.CUSD,
+      }
+      const deployMultiTokenToken = await deployMultiToken(true, body, 'https://alfajores-forno.celo-testnet.org/')
+      expect(deployMultiTokenToken).not.toBeNull()
```
[the remark this answers is not in the document]
Comment (jakcinmarina) [reply packages/tatum/src/multiToken/tests/celo.spec.ts:32]: yeah, will fix that

File: packages/tatum-bsc/src/transaction/bsc.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (vlnevyhosteny) [reply packages/tatum-bsc/src/transaction/bsc.ts:62]: Weird :/

**Pull request closed without merging**